video/audio_sample.h

```diff
@@ -94,9 +93,4 @@ uint32_t AudioSample::getSize() {
 	return samples;
 }
 
-uint32_t AudioSample::getDuration() {
```

Comment:
this function is no longer needed, as it was only ever used by the EnhancedSamplesGenerator